extensions/wikia/GlobalNavigation/scripts/GoogleSearchAPI.js

```diff
@@ -35,6 +35,11 @@ $(function () {
 			searchQuery += ' site:' + window.location.hostname;
 		}
 
+		//Manually trigger page view
+		var currentUrlPath = window.location.pathname;
```

Comment:
please move this var declaration on the top of the function to have one var per scope :)